search-service/src/test/kotlin/com/egm/stellio/search/config/TestContainersConfiguration.kt

```diff
@@ -0,0 +1,45 @@
+package com.egm.stellio.search.config
+
+import com.egm.stellio.shared.config.TestContainers
+import io.r2dbc.spi.ConnectionFactories
+import io.r2dbc.spi.ConnectionFactory
+import io.r2dbc.spi.ConnectionFactoryOptions
+import org.springframework.boot.test.context.TestConfiguration
+import org.springframework.context.annotation.Bean
+
+@TestConfiguration
+class TestContainersConfiguration {
+
+    private val DB_NAME = "stellio_search"
+    private val DB_USER = "stellio_search"
+    private val DB_PASSWORD = "stellio_search_db_password"
+
+    object SearchServiceTestContainers : TestContainers("postgres", 5432) {
+
+        fun getPostgresqlHost(): String {
+            return instance.getServiceHost(serviceName, servicePort)
+        }
+
+        fun getPostgresqlPort(): Int {
+            return instance.getServicePort(serviceName, servicePort)
+        }
+
+        fun getPostgresqlUri(): String {
+            return "jdbc:postgresql://" + getPostgresqlHost() + ":" + getPostgresqlPort() + '/'
+        }
+    }
+
+    @Bean
+    fun configuration(): ConnectionFactory {
```

Comment:
The name of this method should be connectionFactory, right?

Reply:
It's more significant yes.